load-tests/README.md

```diff
@@ -0,0 +1,28 @@
+# Locust based load tests for RH-Che/CReW
+
+### Prepare environment
+
+It is recommended to do python-pip installations in a separate python virtual-env
+
+* Installing dependencies:
+    ```
+    yum install python-pip --assumeyes \
+    pip install locust \
+    pip install websocket-client
+    ```
+
+### Running the tests
+
+```
+cd rh-che/load-tests && \
```

Comment:
Is it needed to have ``&& \`` here?

Reply:
Theoretically the `&&` could be at the start of the next line, but I made it so that you can just select the command block, paste it into the terminal and it should execute without an error.